src/types.ts

```diff
@@ -83,6 +83,8 @@ export interface CreateSelectorOptions<
    */
   inputStabilityCheck?: StabilityCheckFrequency
 
+  noopCheck?: StabilityCheckFrequency
```

Comment:
we should probably rename the type to CheckFrequency, since this isn't a stability check

Reply:
Good idea!

Reply:
How about `DevCheckFrequency` or `DevModeCheckFrequency` , `CheckFrequency` seems a little generic, what do you think?

Reply:
DevCheckFrequency is fine by me

Reply:
Yep, I was literally thinking the same thing reading the current source before I got to this PR!